config/tf_modules/aws-cloudfront-distribution/variables.tf

```diff
@@ -23,7 +23,7 @@ variable "s3_log_bucket_name" {
 
 variable "default_page_file" {
   type    = string
-  default = null
+  default = "index.html"
```

Comment:
How does this default interact with the default in the yaml? Does one override the other? Ideally we would have the default in one place only.

Reply:
You're right-- this is a useless default which always get overwritten by the default in the yaml, I'll remove them now